### PR TITLE
 fixing overwrites from merge

### PR DESCRIPTION
**Pull request by parmitam**
Missed some changes when merging fluent_udf branch to master.

### CODE REVIEW

**Comment by parmitam** [comment]
we missed out a few things in the merge from the fluent_udf branch.
with these changes we should be good to go.

File: myria/test/test_query.py [hunk 1]
```diff
@@ -190,16 +196,17 @@ def test_submit_plan(self):
 
     def test_submit_program(self):
         with HTTMock(local_mock):
-            program = 'COMPLETE_IMMEDIATELY'
+            program = 'CI = empty(i:int);\nstore(CI, CI);'
```
Comment (BrandonHaynes) [inline myria/test/test_query.py]: Why was this rename necessary?  I forget.

Comment (parmitam) [inline myria/test/test_query.py]: I don't remember :(
I think we might be fine without  it -- I merged the tests  file.

**Comment by parmitam** [comment]
These changes look good. What is the Raco fix?
